### PR TITLE
Added multiselect combobox feature support into React SDK

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

**Review by paulgagnon-pega — DISMISSED** [review]

File: packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+  const displayFieldMeta = listType !== 'associated' ? getDisplayFieldsMetaData(dataConfig.columns) : null;
+  const getCaseListBasedOnParamsDebounced: any = useRef();
+  const pConn = getPConnect();
+  const contextName = pConn.getContextName();
+  const listActions = pConn.getListActions();
+  const dataApiObj = useRef();
+
+  const icon = <CheckBoxOutlineBlankIcon fontSize='small' />;
+  const checkedIcon = <CheckBoxIcon fontSize='small' />;
```
Comment (tumms2021389) [inline packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx:118]: You can move constants out of react component

File: packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+  // main search function trigger
+  const getCaseListBasedOnParams = async (searchText, group, selectedRows, currentItemsTree, isTriggeredFromSearch = false) => {
+    if (referenceList && referenceList.length > 0) {
+      selectedRows = await listActions.getSelectedRows(true);
+
+      selectedRows =
+        selectedRows.length > 0
```
Comment (tumms2021389) [inline packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx:126]: I dont understand this ternary condition clearly, Why you are checking the length here

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx (hunk 2 above)
Comment (4manasa) [reply packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx:126]: I'm iterating through the selectedRows to create a map in the expected format for the combobox component. Before the iteration, I'm checking if selectedRows contains any records by verifying its length.

File: packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx (hunk 1 above)
Comment (4manasa) [reply packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx]: Done

File: packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx (hunk 2 above)
Comment (tumms2021389) [reply packages/react-sdk-components/src/components/field/Multiselect/Multiselect.tsx:126]: If selectedRows is already an array then you can directly run .map method on it, you dont have to check the length

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

**Review by tumms2021389 — APPROVED** [review]
Changes looks good to me

**Pull request merged**